### ISSUE TITLE
Invalid API Method

### ISSUE THREAD

**Issue by greebo3** [Gufran/authorize-net-cim-api#2]
Hi ,

From the example:
$api = Authorize::createCustomerProfile()
        ->setFirstName('gufran')
        ->setCity('someCity')
        ->setCardNumber('1234567890123456');

I am receiving the following error when Invalid API Method [Gufran\AuthNet\Entities\RequestMethods\CreateCustomerProfile] Unable to make request

**Comment by greebo3** [comment]
Hi Gufran, Any ideas I could use to bug check this?

**Comment by N30** [comment]
Yeah, his documentation is typing the wrong syntax from the repository name to the methods and everything, CreateCustomerProfile doesn't exist, it is CreateProfile... You should look at the src/Entities/RequestMethods after that cause you will continue to get many errors!
